File: MyApp/app/src/main/java/com/example/garai/starwars/MainActivity.java
package com.example.garai.starwars;

import android.content.Intent;
import android.os.Build;
import android.provider.Settings;
import android.support.annotation.RequiresApi;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.view.View;
import android.widget.ImageButton;
import android.widget.ImageView;
import android.widget.RelativeLayout;
import android.widget.TextView;
import android.widget.Toast;

import org.json.JSONException;
import org.json.JSONObject;

public class MainActivity extends AppMenuActivity {

    Globals globals;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        globals = (Globals) this.getApplication();

        RelativeLayout layout = (RelativeLayout) findViewById(R.id.activity_main);

        getUserTheme(layout);

        getSupportActionBar().setDisplayShowHomeEnabled(true);
        getSupportActionBar().setIcon(R.drawable.logo_menu);

        moveInputPage();

    }


    /**
     * startボタンが押されたとき
     */
    protected void moveInputPage() {

        final ImageButton button = (ImageButton) findViewById(R.id.imageButton);
        button.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {

                checkUserInfo();

            }
        });
    }

    @Override
    protected void onActivityResult(int requestCode, int resultCode, Intent intent) {

        RelativeLayout layout = (RelativeLayout) findViewById(R.id.activity_main);
        changeBackGround(layout);

    }

}

File: MyApp/app/src/main/java/com/example/garai/starwars/MainActivity.kt
package com.example.garai.starwars

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.Gravity
import android.view.View
import android.widget.Button
import android.widget.Toast

import com.example.garai.starwars.R.id.button

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        moveInputPage()
    }


    /**
     * startボタンが押されたとき
     */
    protected fun moveInputPage() {
        val button = findViewById(R.id.button) as Button
        button.setOnClickListener {
            val intent = Intent(application, SecondActivity::class.java)
            startActivity(intent)
        }
    }
}
